-arlt

```diff
@@ -0,0 +1,76 @@
+diff --git a/config/def.yaml b/config/def.yaml
```

Comment:
What is this file? 

It looks like someone ran the output of `git diff` into a file named `-arlt` and included it as part of the commit.

Is this something required by AWS? I'm not sure about this. If this was checked-in by mistake, let's delete `-arlt` from the branch.

Reply:
Removed.